openshift/template.yaml

```diff
@@ -221,6 +221,20 @@ objects:
               name: web
               ports:
                 - containerPort: ${{GT_APP_PORT}}
+              readinessProbe:
+                exec:
+                  command:
+                    - cat
```

Comment:
Does healthy mean "/tmp/healthy file exists" ? Otherwise what is the indication of healthy and unhealthy?

Reply:
As per https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command

Reply:
That looks like a live/ready/ness probe specific for busybox only.